scripts/validate-app-templates.js

```diff
@@ -13,6 +13,7 @@ const appTemplates = require('../lib/app-templates');
 const validateAppTemplate = (template, rootTmpDir) => {
   //const appDir = path.resolve(rootTmpDir, template);
   const zapierCmd = path.resolve(__dirname, '../zapier.js');
+  const extraCmd = (template === 'babel') ? ' && rm -rf lib && node_modules/babel-cli/bin/babel.js src --out-dir lib' : '';
```

Comment:
+1. I know we made that folder and it should be totally fine, but users do [weird things](https://xkcd.com/1172/). I think we can safely re-run babel into an existing directory and it'll update properly.